.env.prd

```diff
@@ -0,0 +1,12 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=Production
+# Change the following values for local development.
+SECRET_KEY=b'I\xfb\xbd\x0cA\xff\xd7u\xee\x81\x8a\x13Ga\x1a\xc0'
+#Trello Keys
+BOARDID = '608a91fd8a30ef4966d68da1'
```

Comment:
Note that the .env files are not Python. The correct way to write this is `BOARDID=608a91fd8a30ef4966d68da1`

python-dotenv strips the spaces and quotes but if you want to run `docker run --env-file .env my-image` then you need to follow that format strictly.